cmd/docs.go

```diff
@@ -34,7 +35,7 @@ func doGenRstDocs(cmd *cobra.Command, args []string) {
 	rootCmd.DisableAutoGenTag = true
 	err := doc.GenReSTTreeCustom(rootCmd, outDir, filePrepender, linkHandler)
 	if err != nil {
-		fmt.Println("ERROR:", err)
+		logrus.Error("ERROR:", err)
 		os.Exit(1)
 	}
```

Comment:
Where I see the value in this commit is that you seem to have spotted all places where we still use a legacy way to exit on error.
All these editions would need to be refactored as below:
```suggestion
subcommands.DieNotNil(doc.GenReSTTreeCustom(rootCmd, outDir, filePrepender, linkHandler))
```

Now, here is the thing: it takes time, so should ultra-low priority.

Reply:
It seems out of scope of this one as well
However, would be great if we start to track those needs down.
WDYT? Could we use issues for that?

Reply:
Don't sweat the "docs" cmd stuff. that exists for us to create documentation in docs.fio.

Reply:
I mean, instead of using `logrus` which its fancy formatting, I'd better use a function we already introduced a while ago to handle errors.

But seriously, I would do neither of the above.
We have more important stuff to do.
That's why our new and old code styles are organically interwoven in many places, and we are all good with that.